src/linesearch/line_model.jl

```diff
@@ -38,8 +38,11 @@ end
 
     ϕ(t) := f(x + td).
 """
-obj(f :: LineModel, t :: Float64) = obj(f.nlp, f.x + t * f.d)
-
+#obj(f :: LineModel, t :: Float64) = obj(f.nlp, f.x + t * f.d)
```

Comment:
Since we have version control, there's no need to comment out code, just remove it.